assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -7663,9 +7663,11 @@ private void post_save_assignment(RunData data, String postOrSave) {
             if (useReviewService
             		&& contentReviewService != null 
             		&& "TurnitinOC".equals(contentReviewService.getServiceName())
+            		&& "false".equalsIgnoreCase((String) serverConfigurationService.getString("turnitin.option.internet_check.default", ""))
```

Comment:
you should check that the variables are being set correctly, not the default settings

Reply:
checkInstitution and checkInternet?